README.md

```diff
@@ -248,9 +248,38 @@ NOTES:
 
 * in order for the metrics to be exported, you also will need to create an instance of CommonFactory
 * common JVM metrics will also be exported alongside common service metrics
+* some metric labels are enumerations (`th2_type`: `MESSAGE_GROUP`, `EVENT`, `<customTag>`;`message_type`: `RAW_MESSAGE`, `MESSAGE`)
+
+ABSTRACT METRICS:
+* th2_rabbitmq_message_size_publish_bytes (`th2_pin`, `th2_type`, `exchange`, `routing_key`): quantity of published message bytes to RabbitMQ. The message is meant any message transferred via RabbitMQ, it can contain th2 batch message or event or other content
```

Comment:
```suggestion
* th2_rabbitmq_message_size_publish_bytes (`th2_pin`, `th2_type`, `exchange`, `routing_key`): number of published message bytes to RabbitMQ. The message is meant for any message transferred via RabbitMQ, it may contain th2 batch message or event or other content
```